platform-sdk/consensus-otter-docker-app/src/testFixtures/java/org/hiero/consensus/otter/docker/app/platform/ConsensusNodeManager.java

```diff
@@ -139,6 +140,9 @@ public ConsensusNodeManager(
         final ReservedSignedState initialState = reservedState.state();
         final MerkleNodeState state = initialState.get().getState();
 
+        // Set active the roster
+        RosterUtils.setActiveRoster(state, activeRoster, DEFAULT_PLATFORM_STATE_FACADE.roundOf(state) + 1);
```

Comment:
I'd prefer that we avoid introducing PlatformFacade here. It will be removed (hopefully very soon), and we should not use it in new code IMO.